src/scenario.h

```diff
@@ -72,6 +72,17 @@ typedef struct {
 	uint32 scenario_count;		// 0x0C
 } rct_scenario_scores_header;
 
+typedef enum {
+	SCENARIO_SOURCE_RCT1_CLASSIC,
+	SCENARIO_SOURCE_RCT1_CORKSCREW,
```

Comment:
I think you should either use the abbreviation AA or the full name: Added Attractions... same with the other packs.